### PR TITLE
(feat) homepage skeleton

### PR DESCRIPTION
**Pull request by kk1024**
Landing page component skeletons

### CODE REVIEW

File: client/src/components/Room/VideoPlayer.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```
Comment (sorbae) [inline client/src/components/Room/VideoPlayer.jsx]: Not required.

File: client/src/components/RoomList.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```
Comment (sorbae) [inline client/src/components/RoomList.jsx]: Bye.

File: client/src/components/Room/VideoPlayer.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+let VideoPlayer = (props) => (
+	<div>
+		<p>VideoPlayer</p>
+	</div>
+)
+
+export default VideoPlayer = VideoPlayer;
```
Comment (sorbae) [inline client/src/components/Room/VideoPlayer.jsx]: Please adhere to airbnb style.

File: client/src/components/RoomListEntry.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```
Comment (sorbae) [inline client/src/components/RoomListEntry.jsx]: Bye.

File: client/src/components/Search.jsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```
Comment (sorbae) [inline client/src/components/Search.jsx]: Not required.

File: client/src/components/Sidebar.jsx [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```
Comment (sorbae) [inline client/src/components/Sidebar.jsx]: Not required.

File: client/src/components/Sidebar.jsx [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+
+let Sidebar = (props) => (
+	<div>
```
Comment (nmorrow11) [inline client/src/components/Sidebar.jsx]: indent 2 spaces

File: client/src/components/Sidebar.jsx [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+		<p>Sidebar</p>
+	</div>
```
Comment (nmorrow11) [inline client/src/components/Sidebar.jsx]: indent 2 spaces

File: client/src/components/index.jsx [hunk 9]
```diff
@@ -1 +1,26 @@
...
+	}
+
+	render() {
+		return (
+			<div>
+				<h1>Fam.ly</h1>
+				<Search />
+				<RoomList rooms={this.state.roomList}/>
+				<Sidebar />
+			</div>)
```
Comment (sorbae) [inline client/src/components/index.jsx]: Move parenthesis down to the next line.

File: client/src/components/Room/VideoPlayer.jsx (hunk 3 above)
Comment (aricalves) [reply client/src/components/Room/VideoPlayer.jsx]: get him frisbee

**Pull request merged**